apps/member-profile/.env.example

```diff
@@ -11,6 +11,8 @@ STUDENT_PROFILE_URL=http://localhost:3000
 # Optional for development, but won't be able to run certain features...
 
 # CRUNCHBASE_BASIC_API_KEY=
+# GITHUB_OAUTH_CLIENT_ID=
+# GITHUB_OAUTH_CLIENT_SECRET=
```

Comment:
I added these to the `.env.example` so that contributors know about these environment variables.